lib/sql/query-insert.js

```diff
@@ -43,9 +43,6 @@ module.exports = class Insert extends RowStreamQuery {
       }
       preppedData[i] = validated.value
       for (var resultKey in validated.value) {
-        if (!dao.ddl[resultKey] || !(resultKey in validator)) {
-          continue
-        }
```

Comment:
🔴 Since we `allowUnknown` during validation, this `if` guards against unknown props.

Reply:
https://github.com/chrisdickinson/ormnomnom/pull/41/files#diff-9bf5f49294bc588915dbeeeb016fb6ecR35 i believe covers that concern